WebDriverAgentLib/Routing/FBXCElementSnapshotWrapper.m

```diff
@@ -94,12 +94,19 @@ - (BOOL)isSelected
 }
 
 #if !TARGET_OS_OSX
+
 - (BOOL)hasFocus
 {
   return self.snapshot.hasFocus;
 }
+
 #endif
 
+- (BOOL)hasKeyboardFocus
+{
+  return self.snapshot.hasKeyboardFocus;
```

Comment:
I assume internally hasFocus and hasKeyboardFocus are the same, aren't they?

Reply:
Hm, the result was different in iOS. I haven't checked on tvOS.
For example when i selected a search bar (then keyboard appeared), the hasFocus was false while the hasKeyboardFocus was true for the element

Reply:
ok, how about tvOS then? Does this attribute make any sense there?
Also, the hasKeyboardFocus property does not seem to be public. I am not sure we want to expose it as Apple may change/remove it unexpectedly and then things gonna go complicated on our side

Reply:
tvOS case was opposit. A focused element had hasFocused true but hasKeyboardFocus was false. (tested only simulator tho)

Reply:
hm, yea, `hasKeyboardFocus` is a private header only for now. okay, lets close this pr. hopefully the hasFocus will work in ios as well.

Reply:
btw, the `selected` is also not clear. It was also always false in my some checks with focus. interesting..